.cirrus.yml

```diff
@@ -213,6 +215,7 @@ task:
   << : *DEPENDS_SDK_CACHE_TEMPLATE
   depends_sources_cache:
     folder: "depends/sources"
+    fingerprint_script: echo $SOURCES_CACHE_FINGERPRINT_KEY
```

Comment:
why not `git log -1 ./depends`?

Reply:
Should we invoke it within a Docker image, i.e., with `DOCKER_EXEC`?

Reply:
It should work as long as `BASE_TEMPLATE` is run before `git` is called

Reply:
Thanks! Done in the recent push.

Reply:
@MarcoFalke 
> why not `git log -1 ./depends`?

It works fine for `depends_sources_cache`, but not for `depends_built_cache` as the the actual content of the latter depends on the `DEP_OPTS` variable.

Reply:
> It works fine for `depends_sources_cache`, but not for `depends_built_cache` as the the actual content of the latter depends on the `DEP_OPTS` variable.

Addressed in the recent push. OP has been updated.